lmod/modulerc_beluga

```diff
@@ -1,12 +1,2 @@
-#%Module
-
-# default versions
-module-version StdEnv/2018.3 default
-module-version boost/1.68.0 default
-module-version boost-mpi/1.68.0 default
-module-version imkl/2018.3.222 default
-module-version intel/2018.3 default
-module-version openmpi/3.1.2 default
```

Comment:
will removing this not break anything when people load modules under StdEnv/2018.3?

Reply:
Mmm, potentially indeed. But keeping them risks being an issue in the future, no ?